### DISCUSSION TITLE
`.local/share/nvim/lazy/lazy.nvim/lua/lazy/manage/git.lua:58` attempt to index local 'range' (a nil value)

### DISCUSSION THREAD

**Discussion by dzintars** [nvim-tree/nvim-tree.lua#2613] [General]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/547420/23441455-916e-45b4-94af-ef58bbde3e0d)
Which plugin issue this might be? Lazy's or Tree's?

NVIM v0.10.0-dev-3079fa1
Build type: RelWithDebInfo
LuaJIT 2.1.1695653777

Just updated Nvim-tree and this got thrown at me.

**Comment by dzintars** [comment]
Solved. Looks like those was my outdated configs.
Changed `version = 'nightly',` to `version = 'v0.*',` in Lazy config for Nvim-tree loading.

**Reply by DartMitai** [reply]
thanks bro
